Gui/ProjectGuiSerialization.h

```diff
@@ -80,7 +80,9 @@ GCC_DIAG_ON(unused-parameter)
 #define PROJECT_GUI_SERIALIZATION_SCRIPT_EDITOR 9
 #define PROJECT_GUI_SERIALIZATION_MERGE_BACKDROP 10
 #define PROJECT_GUI_SERIALIZATION_INTRODUCES_PYTHON_PANELS 11
-#define PROJECT_GUI_SERIALIZATION_VERSION PROJECT_GUI_SERIALIZATION_INTRODUCES_PYTHON_PANELS
+#define PROJECT_GUI_SERIALIZATION_INTRODUCES_MINIMIZED_PANELS 12
```

Comment:
can you please make a single version bump instead of two? It reduces code clutter